src/main/scala/com/lightbend/kafka/scala/streams/typesafe/TSKType.scala

```diff
@@ -0,0 +1,11 @@
+package com.lightbend.kafka.scala.streams.typesafe
+
+import scala.language.higherKinds
+
+/** A base type for all base stream classes in the typesafe API.
+  *
+  * @author Santiago Saavedra (ssaavedra@openshine.com)
```

Comment:
I know we don't have a contribution policy defined for this project, but in general we shouldn't need author attributions in the codebase because the community will inherit its maintenance.

Reply:
I understand that (actually it was an automatic comment left over from the IDE) but if you want contributors to follow any policy, there should be one first :)